datafoundation/templates/storagecluster.go

```diff
@@ -31,6 +31,34 @@ const (
 	backingStorageClassName = "default-ocs-storage-class"
 )
 
+var StorageClusterMetrics = []string{
+	// "job:ceph_versions_running:count",
+	// "job:ceph_pools_iops_bytes:total",
+	// "job:ceph_pools_iops:total",
+	// "job:ceph_osd_metadata:count",
+	// "ceph_health_status",
+	// "ceph_cluster_total_used_raw_bytes",
+	// "ceph_cluster_total_bytes",
+}
+
+var StorageClusterAlerts = []string{
+	// "CephMdsMissingReplicas",
+	// "CephMgrIsAbsent",
+	// "CephMgrIsMissingReplicas",
+	// "CephNodeDown",
+	// "CephClusterErrorState",
+	// "CephClusterWarningState",
+	// "CephOSDVersionMismatch",
+	// "CephMonVersionMismatch",
+	// "CephOSDFlapping",
+	// "CephOSDDiskNotResponding",
+	// "CephOSDDiskUnavailable",
+	// "CephDataRecoveryTakingTooLong",
+	// "CephPGRepairTakingTooLong",
+	// "CephMonQuorumAtRisk",
+	// "CephMonQuorumLost",
+}
```

Comment:
Why are these here? Let's keep everything where it was until we have a mechanism to allow different offering to provide their own metrics dynamically (outside the scope of this PR). I don't like the fact, that with this change, generalized code needs to statically import the storagecluster template.  Not to mention that this has nothing to do with the storagecluster itself.